napari_console/_tests/test_qt_console.py

```diff
@@ -46,3 +46,26 @@ def mock_get_ipython():
         console = QtConsole(viewer)
         qtbot.addWidget(console)
         assert console.kernel_client is None
+
+
+def test_console_focus_proxy(qtbot, make_test_viewer):
+    """Test setting/clearing focus on a QtConsole sets/clears focus on the underlying QTextEdit"""
+    viewer = make_test_viewer()
+    console = QtConsole(viewer)
+    qtbot.addWidget(console)
```

Comment:
why create a separate console instead of `viewer.window._qt_viewer.console`?

Reply:
Good question - I just copied that part of the test from the above tests but can try this instead.